backend/backend/api/routes.py

```diff
@@ -346,4 +353,18 @@ async def github_comment(repo, pr_number, request):
         with gr.Column():
             github_comment_output = gr.Textbox(label="Response")
     github_comment_button.click(github_comment, inputs=[
-                                repo_input, pr_number_input, request_input], outputs=github_comment_output)
+                                repo_input, pr_number_input, request_input],
+                                outputs=github_comment_output)
+
+    gr.Markdown("## GitHub PR Review")
+    with gr.Row():
+        with gr.Column():
+            repo_input = gr.Textbox(label="Enter the repository here")
+            pr_number_input = gr.Textbox(
+                label="Enter the pull request number here")
+            github_comment_button = gr.Button("Review PR")
+        with gr.Column():
+            github_comment_output = gr.Textbox(label="Response")
+    github_comment_button.click(github_pr, inputs=[
+                                repo_input, pr_number_input],
```

Comment:
The inputs to the github_comment_button.click method have been changed. The request_input is no longer included in the list of inputs, and only repo_input and pr_number_input are passed as arguments.